Description: Avoid test that fails randomly based on random seed.
 See https://github.com/pydata/xarray/issues/6504
Bug-Origin: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1013939
Author: Alastair McKinstry <mckinstry@debian.org>
Last-Updated: 2022-080-23
Forwarded: no

--- a/xarray/tests/test_weighted.py
+++ b/xarray/tests/test_weighted.py
@@ -657,6 +657,8 @@
         ([2, 3, 4, 5], [0, 1, 2, 3], 3.8),  # no weights for coord a == 0 or 1
     ],
 )
+
+@pytest.mark.xfail
 def test_weighted_operations_nonequal_coords(
     coords_weights: Iterable[Any],
     coords_data: Iterable[Any],
